src/main/webapp/map.js

```diff
@@ -89,8 +92,29 @@ function codeAddress() {
   
  function callback(results, status) {
   if (status == google.maps.places.PlacesServiceStatus.OK) {
-    for (var i = 0; i < results.length; i++) {
+    // Display markers for Explore tab
+    if (document.URL.includes("store.html")) {
+      for (var i = 0; i < results.length; i++) {
+        createMarker(results[i]);
+      }
+    // Calculate distance for Search for a drink tab
+    } else {
+      coffeeShopInfo = [];
+      for (var i = 0; i < results.length; i++) {
+        var distance = haversine_distance(results[i].geometry.location.lat(),
```

Comment:
Create an issue to update the design doc with a note you're using this algorithm.